software-requests/gitlab-runner/requestDetails.md

```diff
@@ -0,0 +1,76 @@
+# GitLab-Runner ITAM Request 
+https://docs.gitlab.com/runner/
+
+## General Product Functionality
+Pulls specificed GitLab repositories for changes and runs a series of 'jobs' and commands defined in that repository within an isolated enviornment defined by the commands. Typically those commands will consist of compile, testing, packaging and deploying stages. It will send the results of those 'jobs' and commands back to GitLab.
+
```

Comment:
We should make mention of being able to leverage GitLabs Auto DevOps functionality https://docs.gitlab.com/ee/topics/autodevops/ to help devs stand up fully featured CI/CD pipelines with minimal work.